Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN echo "<source>\n\
   azure_container                   fluentd\n\
   auto_create_container             true\n\
   path                              logs/\n\
+  azure_msi_client_id               F533561D-51E3-4F73-BE9F-8DD915ACDD06\n\
```

Comment:
This prob should be an env var and not a hard-coded value

Reply:
Removed this line. We don't need it.